nopolinfo #0
location: FailureLocation{className='redis.clients.jedis.tests.ConnectionTest', failingMethods=[redis.clients.jedis.tests.ConnectionTest#getErrorAfterConnectionReset], erroringMethods=[redis.clients.jedis.tests.ConnectionTest#checkCloseable, redis.clients.jedis.tests.ConnectionTest#connectIfNotConnectedWhenSettingTimeoutInfinite], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='org.junit.ComparisonFailure', failureDetail='expected:<[ERR Protocol error: invalid multibulk length]> but was:<[Failed connecting to host localhost:6379]>', isError=false}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=1, nbErrors=2}
status: NOPATCH
dateEnd: Sun Dec 03 08:08:33 EST 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/310839695/src/main/java], projectClasspath='[Ljava.net.URL;@77c692b4', projectTests=[redis.clients.jedis.tests.ConnectionTest], complianceLevel=8, outputFolder=./, json=false}
exception: null
nbStatements: 31
nbAngelicValues: 23
ignoreStatus: IGNORE_ERRORING
----------



nopolinfo #1
location: FailureLocation{className='redis.clients.jedis.tests.ConnectionTest', failingMethods=[redis.clients.jedis.tests.ConnectionTest#getErrorAfterConnectionReset], erroringMethods=[redis.clients.jedis.tests.ConnectionTest#checkCloseable, redis.clients.jedis.tests.ConnectionTest#connectIfNotConnectedWhenSettingTimeoutInfinite], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='org.junit.ComparisonFailure', failureDetail='expected:<[ERR Protocol error: invalid multibulk length]> but was:<[Failed connecting to host localhost:6379]>', isError=false}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=1, nbErrors=2}
status: NOPATCH
dateEnd: Sun Dec 03 08:09:00 EST 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/310839695/src/main/java], projectClasspath='[Ljava.net.URL;@6b15e68c', projectTests=[redis.clients.jedis.tests.ConnectionTest], complianceLevel=8, outputFolder=./, json=false}
exception: null
nbStatements: 31
nbAngelicValues: 23
ignoreStatus: IGNORE_FAILING
----------



nopolinfo #2
location: FailureLocation{className='redis.clients.jedis.tests.JedisClusterTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.JedisClusterTest#testStableSlotWhenMigratingNodeOrImportingNodeIsNotSpecified, redis.clients.jedis.tests.JedisClusterTest#testClusterFlushSlots, redis.clients.jedis.tests.JedisClusterTest#testClusterKeySlot, redis.clients.jedis.tests.JedisClusterTest#testReturnConnectionOnJedisConnectionException, redis.clients.jedis.tests.JedisClusterTest#testRedisClusterMaxRedirections, redis.clients.jedis.tests.JedisClusterTest#testCloseable, redis.clients.jedis.tests.JedisClusterTest#testLocalhostNodeNotAddedWhen127Present, redis.clients.jedis.tests.JedisClusterTest#testMigrate, redis.clients.jedis.tests.JedisClusterTest#testIfPoolConfigAppliesToClusterPools, redis.clients.jedis.tests.JedisClusterTest#testClusterForgetNode, redis.clients.jedis.tests.JedisClusterTest#testCalculateConnectionPerSlot, redis.clients.jedis.tests.JedisClusterTest#testMovedExceptionParameters, redis.clients.jedis.tests.JedisClusterTest#testRecalculateSlotsWhenMoved, redis.clients.jedis.tests.JedisClusterTest#testThrowAskException, redis.clients.jedis.tests.JedisClusterTest#testAskResponse, redis.clients.jedis.tests.JedisClusterTest#testReadonly, redis.clients.jedis.tests.JedisClusterTest#testRedisHashtag, redis.clients.jedis.tests.JedisClusterTest#testJedisClusterTimeout, redis.clients.jedis.tests.JedisClusterTest#testInvalidStartNodeNotAdded, redis.clients.jedis.tests.JedisClusterTest#testJedisClusterRunsWithMultithreaded, redis.clients.jedis.tests.JedisClusterTest#redis.clients.jedis.tests.JedisClusterTest, redis.clients.jedis.tests.JedisClusterTest#testMigrateToNewNode, redis.clients.jedis.tests.JedisClusterTest#testDiscoverNodesAutomatically, redis.clients.jedis.tests.JedisClusterTest#testThrowMovedException, redis.clients.jedis.tests.JedisClusterTest#testClusterCountKeysInSlot, redis.clients.jedis.tests.JedisClusterTest#testReturnConnectionOnRedirection, redis.clients.jedis.tests.JedisClusterTest#testSetClientName], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}], nbFailures=0, nbErrors=53}
status: NOPATCH
dateEnd: Sun Dec 03 08:13:37 EST 2017
allocatedtime: 120minutes 
passingTime: 4minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/310839695/src/main/java], projectClasspath='[Ljava.net.URL;@38f3dbbf', projectTests=[redis.clients.jedis.tests.JedisClusterTest], complianceLevel=8, outputFolder=./, json=false}
exception: null
nbStatements: 52
nbAngelicValues: 35
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #3
location: FailureLocation{className='redis.clients.jedis.tests.commands.SortedSetCommandsTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.commands.SortedSetCommandsTest#zrange, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zscanMatch, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zrevrange, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zcard, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zrank, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zintertoreParams, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zrangeByLexBinary, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zunionstore, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zrevrangebyscore, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zlexcountBinary, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zrevrangeByLex, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zincrbyWithParams, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zcount, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zrangebyscoreWithScores, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zremrangeByLex, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zremrangeByScore, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zrangebyscore, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zadd, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zaddWithParams, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zscanCount, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zremrangeByRank, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zunionstoreParams, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zrangeWithScores, redis.clients.jedis.tests.commands.SortedSetCommandsTest#infinity, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zrevrank, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zinterstore, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zrevrangebyscoreWithScores, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zrem, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zrevrangeWithScores, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zincrby, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zlexcount, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zrangeByLex, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zremrangeByLexBinary, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zscan, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zrevrangeByLexBinary, redis.clients.jedis.tests.commands.SortedSetCommandsTest#zscore], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=36}
status: NOPATCH
dateEnd: Sun Dec 03 08:16:01 EST 2017
allocatedtime: 118minutes 
passingTime: 2minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/310839695/src/main/java], projectClasspath='[Ljava.net.URL;@5d414943', projectTests=[redis.clients.jedis.tests.commands.SortedSetCommandsTest], complianceLevel=8, outputFolder=./, json=false}
exception: null
nbStatements: 32
nbAngelicValues: 26
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #4
location: FailureLocation{className='redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#ttl, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#renameOldAndNewAreTheSame, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#swapDB, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#pexpireAt, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#persist, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#expireAt, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#scanMatch, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#existsMany, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#dbSize, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#rename, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#unlink, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#expire, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#psetex, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#move, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#dumpAndRestore, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#getDB, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#flushAll, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#exists, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#randomKey, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#scanIsCompleteIteration, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#select, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#setNxExAndGet, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#echo, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#pingWithMessage, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#del, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#pexpire, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#scanCount, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#pttl, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#flushDB, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#scan, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#type, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#touch, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#renamenx, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#keys, redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest#ping], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=35}
status: NOPATCH
dateEnd: Sun Dec 03 08:18:19 EST 2017
allocatedtime: 117minutes 
passingTime: 2minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/310839695/src/main/java], projectClasspath='[Ljava.net.URL;@26db7540', projectTests=[redis.clients.jedis.tests.commands.AllKindOfValuesCommandsTest], complianceLevel=8, outputFolder=./, json=false}
exception: null
nbStatements: 32
nbAngelicValues: 26
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #5
location: FailureLocation{className='redis.clients.jedis.tests.PipeliningTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.PipeliningTest#testEval, redis.clients.jedis.tests.PipeliningTest#pipelineBinarySafeHashCommands, redis.clients.jedis.tests.PipeliningTest#pipelineResponseWithinPipeline, redis.clients.jedis.tests.PipeliningTest#testPipelinedTransactionResponse, redis.clients.jedis.tests.PipeliningTest#testReuseJedisWhenPipelineIsEmpty, redis.clients.jedis.tests.PipeliningTest#testEvalshaKeyAndArg, redis.clients.jedis.tests.PipeliningTest#pipelineResponseWithoutData, redis.clients.jedis.tests.PipeliningTest#testEvalNestedLists, redis.clients.jedis.tests.PipeliningTest#testCloseableWithMulti, redis.clients.jedis.tests.PipeliningTest#pipelineExecShoudThrowJedisDataExceptionWhenNotInMulti, redis.clients.jedis.tests.PipeliningTest#piplineWithError, redis.clients.jedis.tests.PipeliningTest#pipelineSelect, redis.clients.jedis.tests.PipeliningTest#testEvalshaKeyAndArgWithBinary, redis.clients.jedis.tests.PipeliningTest#testDiscardInPipeline, redis.clients.jedis.tests.PipeliningTest#pipelineWithPubSub, redis.clients.jedis.tests.PipeliningTest#pipeline, redis.clients.jedis.tests.PipeliningTest#testEvalsha, redis.clients.jedis.tests.PipeliningTest#canRetrieveUnsetKey, redis.clients.jedis.tests.PipeliningTest#testEvalKeyAndArg, redis.clients.jedis.tests.PipeliningTest#multiWithMassiveRequests, redis.clients.jedis.tests.PipeliningTest#pipelineResponse, redis.clients.jedis.tests.PipeliningTest#multiWithSync, redis.clients.jedis.tests.PipeliningTest#testEvalNestedListsWithBinary, redis.clients.jedis.tests.PipeliningTest#testJedisThowExceptionWhenInPipeline, redis.clients.jedis.tests.PipeliningTest#testSyncWithNoCommandQueued, redis.clients.jedis.tests.PipeliningTest#testResetStateWhenInPipeline, redis.clients.jedis.tests.PipeliningTest#testCloseable, redis.clients.jedis.tests.PipeliningTest#testEvalKeyAndArgWithBinary, redis.clients.jedis.tests.PipeliningTest#testEvalWithBinary, redis.clients.jedis.tests.PipeliningTest#pipelineMultiShoudThrowJedisDataExceptionWhenAlreadyInMulti, redis.clients.jedis.tests.PipeliningTest#pipelineResponseWithData, redis.clients.jedis.tests.PipeliningTest#pipelineDiscardShoudThrowJedisDataExceptionWhenNotInMulti, redis.clients.jedis.tests.PipeliningTest#multi], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=33}
status: NOPATCH
dateEnd: Sun Dec 03 08:20:26 EST 2017
allocatedtime: 116minutes 
passingTime: 2minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/310839695/src/main/java], projectClasspath='[Ljava.net.URL;@67d361fa', projectTests=[redis.clients.jedis.tests.PipeliningTest], complianceLevel=8, outputFolder=./, json=false}
exception: null
nbStatements: 28
nbAngelicValues: 21
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #6
location: FailureLocation{className='redis.clients.jedis.tests.commands.ClusterCommandsTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.commands.ClusterCommandsTest#redis.clients.jedis.tests.commands.ClusterCommandsTest, redis.clients.jedis.tests.commands.ClusterCommandsTest#clusterAddSlots, redis.clients.jedis.tests.commands.ClusterCommandsTest#clusterInfo, redis.clients.jedis.tests.commands.ClusterCommandsTest#clusterSetSlotNode, redis.clients.jedis.tests.commands.ClusterCommandsTest#clusterSetSlotImporting, redis.clients.jedis.tests.commands.ClusterCommandsTest#testClusterHardReset, redis.clients.jedis.tests.commands.ClusterCommandsTest#clusterGetKeysInSlot, redis.clients.jedis.tests.commands.ClusterCommandsTest#clusterSlots, redis.clients.jedis.tests.commands.ClusterCommandsTest#testClusterSoftReset, redis.clients.jedis.tests.commands.ClusterCommandsTest#clusterDelSlots, redis.clients.jedis.tests.commands.ClusterCommandsTest#clusterMeet, redis.clients.jedis.tests.commands.ClusterCommandsTest#clusterSetSlotMigrating, redis.clients.jedis.tests.commands.ClusterCommandsTest#clusterNodes], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='java.lang.NullPointerException', failureDetail='null', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='java.lang.NullPointerException', failureDetail='null', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='java.lang.NullPointerException', failureDetail='null', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='java.lang.NullPointerException', failureDetail='null', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='java.lang.NullPointerException', failureDetail='null', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='java.lang.NullPointerException', failureDetail='null', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='java.lang.NullPointerException', failureDetail='null', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='java.lang.NullPointerException', failureDetail='null', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='java.lang.NullPointerException', failureDetail='null', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='java.lang.NullPointerException', failureDetail='null', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='java.lang.NullPointerException', failureDetail='null', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='java.lang.NullPointerException', failureDetail='null', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}], nbFailures=0, nbErrors=25}
status: NOPATCH
dateEnd: Sun Dec 03 08:22:32 EST 2017
allocatedtime: 115minutes 
passingTime: 2minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/310839695/src/main/java], projectClasspath='[Ljava.net.URL;@354aaf03', projectTests=[redis.clients.jedis.tests.commands.ClusterCommandsTest], complianceLevel=8, outputFolder=./, json=false}
exception: null
nbStatements: 55
nbAngelicValues: 39
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #7
location: FailureLocation{className='redis.clients.jedis.tests.commands.BinaryValuesCommandsTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#incrWrongValue, redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#setAndGet, redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#mget, redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#msetnx, redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#decrByWrongValue, redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#getSet, redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#incrBy, redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#setIfExistAndGet, redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#setex, redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#setFailIfNotExistAndGet, redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#incrByWrongValue, redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#decrBy, redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#substr, redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#strlen, redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#setAndExpireMillis, redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#setnx, redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#mset, redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#incr, redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#setAndExpire, redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#setIfNotExistAndGet, redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#decr, redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#append, redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#decrWrongValue, redis.clients.jedis.tests.commands.BinaryValuesCommandsTest#setNxExAndGet], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=24}
status: NOPATCH
dateEnd: Sun Dec 03 08:24:15 EST 2017
allocatedtime: 114minutes 
passingTime: 1minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/310839695/src/main/java], projectClasspath='[Ljava.net.URL;@268206aa', projectTests=[redis.clients.jedis.tests.commands.BinaryValuesCommandsTest], complianceLevel=8, outputFolder=./, json=false}
exception: null
nbStatements: 32
nbAngelicValues: 26
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #8
location: FailureLocation{className='redis.clients.jedis.tests.commands.StringValuesCommandsTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.commands.StringValuesCommandsTest#setex, redis.clients.jedis.tests.commands.StringValuesCommandsTest#strlen, redis.clients.jedis.tests.commands.StringValuesCommandsTest#decrByWrongValue, redis.clients.jedis.tests.commands.StringValuesCommandsTest#decr, redis.clients.jedis.tests.commands.StringValuesCommandsTest#incrLargeNumbers, redis.clients.jedis.tests.commands.StringValuesCommandsTest#incrWrongValue, redis.clients.jedis.tests.commands.StringValuesCommandsTest#append, redis.clients.jedis.tests.commands.StringValuesCommandsTest#incrByFloatWrongValue, redis.clients.jedis.tests.commands.StringValuesCommandsTest#setAndGet, redis.clients.jedis.tests.commands.StringValuesCommandsTest#incr, redis.clients.jedis.tests.commands.StringValuesCommandsTest#incrByWrongValue, redis.clients.jedis.tests.commands.StringValuesCommandsTest#mset, redis.clients.jedis.tests.commands.StringValuesCommandsTest#msetnx, redis.clients.jedis.tests.commands.StringValuesCommandsTest#incrReallyLargeNumbers, redis.clients.jedis.tests.commands.StringValuesCommandsTest#mget, redis.clients.jedis.tests.commands.StringValuesCommandsTest#incrBy, redis.clients.jedis.tests.commands.StringValuesCommandsTest#psetex, redis.clients.jedis.tests.commands.StringValuesCommandsTest#substr, redis.clients.jedis.tests.commands.StringValuesCommandsTest#decrBy, redis.clients.jedis.tests.commands.StringValuesCommandsTest#getSet, redis.clients.jedis.tests.commands.StringValuesCommandsTest#decrWrongValue, redis.clients.jedis.tests.commands.StringValuesCommandsTest#incrByFloat, redis.clients.jedis.tests.commands.StringValuesCommandsTest#setnx], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=23}
status: NOPATCH
dateEnd: Sun Dec 03 08:25:56 EST 2017
allocatedtime: 113minutes 
passingTime: 1minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/310839695/src/main/java], projectClasspath='[Ljava.net.URL;@13c4af55', projectTests=[redis.clients.jedis.tests.commands.StringValuesCommandsTest], complianceLevel=8, outputFolder=./, json=false}
exception: null
nbStatements: 32
nbAngelicValues: 26
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #9
location: FailureLocation{className='redis.clients.jedis.tests.commands.HashesCommandsTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.commands.HashesCommandsTest#hscan, redis.clients.jedis.tests.commands.HashesCommandsTest#hexists, redis.clients.jedis.tests.commands.HashesCommandsTest#testBinaryHstrLen, redis.clients.jedis.tests.commands.HashesCommandsTest#hset, redis.clients.jedis.tests.commands.HashesCommandsTest#testHstrLen_EmptyHash, redis.clients.jedis.tests.commands.HashesCommandsTest#hdel, redis.clients.jedis.tests.commands.HashesCommandsTest#testHstrLen, redis.clients.jedis.tests.commands.HashesCommandsTest#hincrBy, redis.clients.jedis.tests.commands.HashesCommandsTest#hgetAll, redis.clients.jedis.tests.commands.HashesCommandsTest#hvals, redis.clients.jedis.tests.commands.HashesCommandsTest#hkeys, redis.clients.jedis.tests.commands.HashesCommandsTest#hmset, redis.clients.jedis.tests.commands.HashesCommandsTest#hmget, redis.clients.jedis.tests.commands.HashesCommandsTest#hincrByFloat, redis.clients.jedis.tests.commands.HashesCommandsTest#hgetAllPipeline, redis.clients.jedis.tests.commands.HashesCommandsTest#hsetnx, redis.clients.jedis.tests.commands.HashesCommandsTest#hget, redis.clients.jedis.tests.commands.HashesCommandsTest#hlen, redis.clients.jedis.tests.commands.HashesCommandsTest#hscanCount, redis.clients.jedis.tests.commands.HashesCommandsTest#hscanMatch, redis.clients.jedis.tests.commands.HashesCommandsTest#hsetVariadic], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=21}
status: NOPATCH
dateEnd: Sun Dec 03 08:27:32 EST 2017
allocatedtime: 113minutes 
passingTime: 1minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/310839695/src/main/java], projectClasspath='[Ljava.net.URL;@214e42a1', projectTests=[redis.clients.jedis.tests.commands.HashesCommandsTest], complianceLevel=8, outputFolder=./, json=false}
exception: null
nbStatements: 32
nbAngelicValues: 26
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #10
location: FailureLocation{className='redis.clients.jedis.tests.commands.ClusterScriptingCommandsTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.commands.ClusterScriptingCommandsTest#testJedisClusterException2, redis.clients.jedis.tests.commands.ClusterScriptingCommandsTest#testBinaryScriptKill, redis.clients.jedis.tests.commands.ClusterScriptingCommandsTest#testEvalsha, redis.clients.jedis.tests.commands.ClusterScriptingCommandsTest#testBinaryEval, redis.clients.jedis.tests.commands.ClusterScriptingCommandsTest#testEval2, redis.clients.jedis.tests.commands.ClusterScriptingCommandsTest#testScriptLoadAndScriptExists, redis.clients.jedis.tests.commands.ClusterScriptingCommandsTest#testJedisClusterException, redis.clients.jedis.tests.commands.ClusterScriptingCommandsTest#redis.clients.jedis.tests.commands.ClusterScriptingCommandsTest, redis.clients.jedis.tests.commands.ClusterScriptingCommandsTest#testBinaryScriptFlush, redis.clients.jedis.tests.commands.ClusterScriptingCommandsTest#testBinaryScriptExists], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='java.lang.NullPointerException', failureDetail='null', isError=true}], nbFailures=0, nbErrors=19}
status: NOPATCH
dateEnd: Sun Dec 03 08:29:19 EST 2017
allocatedtime: 112minutes 
passingTime: 1minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/310839695/src/main/java], projectClasspath='[Ljava.net.URL;@dce343c', projectTests=[redis.clients.jedis.tests.commands.ClusterScriptingCommandsTest], complianceLevel=8, outputFolder=./, json=false}
exception: null
nbStatements: 63
nbAngelicValues: 39
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #11
location: FailureLocation{className='redis.clients.jedis.tests.commands.ListCommandsTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.commands.ListCommandsTest#rpop, redis.clients.jedis.tests.commands.ListCommandsTest#rpush, redis.clients.jedis.tests.commands.ListCommandsTest#llenNotOnList, redis.clients.jedis.tests.commands.ListCommandsTest#ltrim, redis.clients.jedis.tests.commands.ListCommandsTest#brpoplpush, redis.clients.jedis.tests.commands.ListCommandsTest#lrange, redis.clients.jedis.tests.commands.ListCommandsTest#lpop, redis.clients.jedis.tests.commands.ListCommandsTest#rpoplpush, redis.clients.jedis.tests.commands.ListCommandsTest#lindex, redis.clients.jedis.tests.commands.ListCommandsTest#llen, redis.clients.jedis.tests.commands.ListCommandsTest#lpush, redis.clients.jedis.tests.commands.ListCommandsTest#lpushx, redis.clients.jedis.tests.commands.ListCommandsTest#lrem, redis.clients.jedis.tests.commands.ListCommandsTest#brpop, redis.clients.jedis.tests.commands.ListCommandsTest#rpushx, redis.clients.jedis.tests.commands.ListCommandsTest#lset, redis.clients.jedis.tests.commands.ListCommandsTest#blpop, redis.clients.jedis.tests.commands.ListCommandsTest#linsert], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=18}
status: NOPATCH
dateEnd: Sun Dec 03 08:30:44 EST 2017
allocatedtime: 112minutes 
passingTime: 1minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/310839695/src/main/java], projectClasspath='[Ljava.net.URL;@59f91ee8', projectTests=[redis.clients.jedis.tests.commands.ListCommandsTest], complianceLevel=8, outputFolder=./, json=false}
exception: null
nbStatements: 32
nbAngelicValues: 26
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #12
location: FailureLocation{className='redis.clients.jedis.tests.commands.ScriptingCommandsTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.commands.ScriptingCommandsTest#evalsha, redis.clients.jedis.tests.commands.ScriptingCommandsTest#evalInt, redis.clients.jedis.tests.commands.ScriptingCommandsTest#scriptLoadBinary, redis.clients.jedis.tests.commands.ScriptingCommandsTest#evalMultiBulkWithBinaryJedis, redis.clients.jedis.tests.commands.ScriptingCommandsTest#evalNestedLists, redis.clients.jedis.tests.commands.ScriptingCommandsTest#evalshaShaNotFound, redis.clients.jedis.tests.commands.ScriptingCommandsTest#scriptExistsBinary, redis.clients.jedis.tests.commands.ScriptingCommandsTest#scriptKill, redis.clients.jedis.tests.commands.ScriptingCommandsTest#scriptExistsWithBrokenConnection, redis.clients.jedis.tests.commands.ScriptingCommandsTest#evalMultiBulk, redis.clients.jedis.tests.commands.ScriptingCommandsTest#evalBulk, redis.clients.jedis.tests.commands.ScriptingCommandsTest#scriptEvalReturnNullValues, redis.clients.jedis.tests.commands.ScriptingCommandsTest#scriptEvalShaReturnNullValues, redis.clients.jedis.tests.commands.ScriptingCommandsTest#scriptLoad, redis.clients.jedis.tests.commands.ScriptingCommandsTest#scriptFlush, redis.clients.jedis.tests.commands.ScriptingCommandsTest#evalshaBinary, redis.clients.jedis.tests.commands.ScriptingCommandsTest#scriptExists, redis.clients.jedis.tests.commands.ScriptingCommandsTest#evalNoArgs], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=18}
status: NOPATCH
dateEnd: Sun Dec 03 08:32:10 EST 2017
allocatedtime: 111minutes 
passingTime: 1minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/310839695/src/main/java], projectClasspath='[Ljava.net.URL;@3c7ca576', projectTests=[redis.clients.jedis.tests.commands.ScriptingCommandsTest], complianceLevel=8, outputFolder=./, json=false}
exception: null
nbStatements: 32
nbAngelicValues: 26
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #13
location: FailureLocation{className='redis.clients.jedis.tests.commands.SetCommandsTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.commands.SetCommandsTest#scard, redis.clients.jedis.tests.commands.SetCommandsTest#sscanCount, redis.clients.jedis.tests.commands.SetCommandsTest#sadd, redis.clients.jedis.tests.commands.SetCommandsTest#smove, redis.clients.jedis.tests.commands.SetCommandsTest#srem, redis.clients.jedis.tests.commands.SetCommandsTest#sscanMatch, redis.clients.jedis.tests.commands.SetCommandsTest#sunionstore, redis.clients.jedis.tests.commands.SetCommandsTest#srandmember, redis.clients.jedis.tests.commands.SetCommandsTest#spop, redis.clients.jedis.tests.commands.SetCommandsTest#sismember, redis.clients.jedis.tests.commands.SetCommandsTest#sinterstore, redis.clients.jedis.tests.commands.SetCommandsTest#sunion, redis.clients.jedis.tests.commands.SetCommandsTest#smembers, redis.clients.jedis.tests.commands.SetCommandsTest#sdiffstore, redis.clients.jedis.tests.commands.SetCommandsTest#sscan, redis.clients.jedis.tests.commands.SetCommandsTest#sdiff, redis.clients.jedis.tests.commands.SetCommandsTest#sinter, redis.clients.jedis.tests.commands.SetCommandsTest#spopWithCount], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=18}
status: NOPATCH
dateEnd: Sun Dec 03 08:33:32 EST 2017
allocatedtime: 111minutes 
passingTime: 1minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/310839695/src/main/java], projectClasspath='[Ljava.net.URL;@60f1d1cb', projectTests=[redis.clients.jedis.tests.commands.SetCommandsTest], complianceLevel=8, outputFolder=./, json=false}
exception: null
nbStatements: 32
nbAngelicValues: 26
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #14
location: FailureLocation{className='redis.clients.jedis.tests.JedisPoolTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.JedisPoolTest#checkConnectionWithDefaultPort, redis.clients.jedis.tests.JedisPoolTest#returnResourceShouldResetState, redis.clients.jedis.tests.JedisPoolTest#getNumActiveReturnsTheCorrectNumber, redis.clients.jedis.tests.JedisPoolTest#checkConnections, redis.clients.jedis.tests.JedisPoolTest#testCloseConnectionOnMakeObject, redis.clients.jedis.tests.JedisPoolTest#checkJedisIsReusedWhenReturned, redis.clients.jedis.tests.JedisPoolTest#checkPoolRepairedWhenJedisIsBroken, redis.clients.jedis.tests.JedisPoolTest#customClientName, redis.clients.jedis.tests.JedisPoolTest#checkCloseableConnections, redis.clients.jedis.tests.JedisPoolTest#startWithUrl, redis.clients.jedis.tests.JedisPoolTest#checkResourceIsCloseable, redis.clients.jedis.tests.JedisPoolTest#securePool, redis.clients.jedis.tests.JedisPoolTest#startWithUrlString, redis.clients.jedis.tests.JedisPoolTest#nonDefaultDatabase, redis.clients.jedis.tests.JedisPoolTest#selectDatabaseOnActivation, redis.clients.jedis.tests.JedisPoolTest#checkPoolOverflow, redis.clients.jedis.tests.JedisPoolTest#testAddObject], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Could not get a resource from the pool', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Could not get a resource from the pool', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Could not get a resource from the pool', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Could not get a resource from the pool', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Could not get a resource from the pool', isError=true}, FailureType{failureName='java.lang.Exception', failureDetail='Unexpected exception, expected<redis.clients.jedis.exceptions.JedisExhaustedPoolException> but was<redis.clients.jedis.exceptions.JedisConnectionException>', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Could not get a resource from the pool', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6380', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Could not get a resource from the pool', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Could not get a resource from the pool', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisException', failureDetail='Error trying to add idle objects', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6380', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Could not get a resource from the pool', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Could not get a resource from the pool', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Could not get a resource from the pool', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Could not get a resource from the pool', isError=true}], nbFailures=0, nbErrors=17}
status: NOPATCH
dateEnd: Sun Dec 03 08:41:11 EST 2017
allocatedtime: 110minutes 
passingTime: 7minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/310839695/src/main/java], projectClasspath='[Ljava.net.URL;@7c7e0c55', projectTests=[redis.clients.jedis.tests.JedisPoolTest], complianceLevel=8, outputFolder=./, json=false}
exception: null
nbStatements: 179
nbAngelicValues: 117
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #15
location: FailureLocation{className='redis.clients.jedis.tests.commands.ClusterBinaryJedisCommandsTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.commands.ClusterBinaryJedisCommandsTest#testGetSlot, redis.clients.jedis.tests.commands.ClusterBinaryJedisCommandsTest#failKeys, redis.clients.jedis.tests.commands.ClusterBinaryJedisCommandsTest#testHmset, redis.clients.jedis.tests.commands.ClusterBinaryJedisCommandsTest#testRpush, redis.clients.jedis.tests.commands.ClusterBinaryJedisCommandsTest#testIncr, redis.clients.jedis.tests.commands.ClusterBinaryJedisCommandsTest#testKeys, redis.clients.jedis.tests.commands.ClusterBinaryJedisCommandsTest#redis.clients.jedis.tests.commands.ClusterBinaryJedisCommandsTest, redis.clients.jedis.tests.commands.ClusterBinaryJedisCommandsTest#testBinaryGetAndSet, redis.clients.jedis.tests.commands.ClusterBinaryJedisCommandsTest#testSadd], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:7379', isError=true}, FailureType{failureName='java.lang.NullPointerException', failureDetail='null', isError=true}], nbFailures=0, nbErrors=17}
status: NOPATCH
dateEnd: Sun Dec 03 08:42:53 EST 2017
allocatedtime: 107minutes 
passingTime: 1minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/310839695/src/main/java], projectClasspath='[Ljava.net.URL;@2289ee66', projectTests=[redis.clients.jedis.tests.commands.ClusterBinaryJedisCommandsTest], complianceLevel=8, outputFolder=./, json=false}
exception: null
nbStatements: 63
nbAngelicValues: 39
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #16
location: FailureLocation{className='redis.clients.jedis.tests.commands.TransactionCommandsTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.commands.TransactionCommandsTest#validateWhenInMulti, redis.clients.jedis.tests.commands.TransactionCommandsTest#testCloseable, redis.clients.jedis.tests.commands.TransactionCommandsTest#transactionResponse, redis.clients.jedis.tests.commands.TransactionCommandsTest#discard, redis.clients.jedis.tests.commands.TransactionCommandsTest#testResetStateWithFullyExecutedTransaction, redis.clients.jedis.tests.commands.TransactionCommandsTest#transactionResponseBinary, redis.clients.jedis.tests.commands.TransactionCommandsTest#testResetStateWhenInMulti, redis.clients.jedis.tests.commands.TransactionCommandsTest#transactionResponseWithinPipeline, redis.clients.jedis.tests.commands.TransactionCommandsTest#testResetStateWhenInMultiWithinPipeline, redis.clients.jedis.tests.commands.TransactionCommandsTest#transactionResponseWithError, redis.clients.jedis.tests.commands.TransactionCommandsTest#testResetStateWhenInWatch, redis.clients.jedis.tests.commands.TransactionCommandsTest#multi, redis.clients.jedis.tests.commands.TransactionCommandsTest#unwatch, redis.clients.jedis.tests.commands.TransactionCommandsTest#select, redis.clients.jedis.tests.commands.TransactionCommandsTest#execGetResponse, redis.clients.jedis.tests.commands.TransactionCommandsTest#watch], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=16}
status: NOPATCH
dateEnd: Sun Dec 03 08:44:14 EST 2017
allocatedtime: 106minutes 
passingTime: 1minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/310839695/src/main/java], projectClasspath='[Ljava.net.URL;@1e3d6f18', projectTests=[redis.clients.jedis.tests.commands.TransactionCommandsTest], complianceLevel=8, outputFolder=./, json=false}
exception: null
nbStatements: 32
nbAngelicValues: 26
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #17
location: FailureLocation{className='redis.clients.jedis.tests.commands.PublishSubscribeCommandsTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.commands.PublishSubscribeCommandsTest#handleClientOutputBufferLimitForSubscribeTooSlow, redis.clients.jedis.tests.commands.PublishSubscribeCommandsTest#binaryPsubscribeMany, redis.clients.jedis.tests.commands.PublishSubscribeCommandsTest#psubscribe, redis.clients.jedis.tests.commands.PublishSubscribeCommandsTest#pubSubNumSub, redis.clients.jedis.tests.commands.PublishSubscribeCommandsTest#subscribeLazily, redis.clients.jedis.tests.commands.PublishSubscribeCommandsTest#pubSubChannels, redis.clients.jedis.tests.commands.PublishSubscribeCommandsTest#binarySubscribeMany, redis.clients.jedis.tests.commands.PublishSubscribeCommandsTest#pubSubNumPat, redis.clients.jedis.tests.commands.PublishSubscribeCommandsTest#binarySubscribe, redis.clients.jedis.tests.commands.PublishSubscribeCommandsTest#binaryPsubscribe, redis.clients.jedis.tests.commands.PublishSubscribeCommandsTest#subscribeMany, redis.clients.jedis.tests.commands.PublishSubscribeCommandsTest#psubscribeMany, redis.clients.jedis.tests.commands.PublishSubscribeCommandsTest#unsubscribeWhenNotSusbscribed, redis.clients.jedis.tests.commands.PublishSubscribeCommandsTest#binarySubscribeLazily, redis.clients.jedis.tests.commands.PublishSubscribeCommandsTest#subscribe, redis.clients.jedis.tests.commands.PublishSubscribeCommandsTest#pubSubChannelWithPingPong], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=16}
status: NOPATCH
dateEnd: Sun Dec 03 08:45:35 EST 2017
allocatedtime: 106minutes 
passingTime: 1minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/310839695/src/main/java], projectClasspath='[Ljava.net.URL;@fba8bb5', projectTests=[redis.clients.jedis.tests.commands.PublishSubscribeCommandsTest], complianceLevel=8, outputFolder=./, json=false}
exception: null
nbStatements: 32
nbAngelicValues: 26
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #18
location: FailureLocation{className='redis.clients.jedis.tests.JedisTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.JedisTest#timeoutConnectionWithURI, redis.clients.jedis.tests.JedisTest#startWithUrl, redis.clients.jedis.tests.JedisTest#shouldReconnectToSameDB, redis.clients.jedis.tests.JedisTest#timeoutConnection, redis.clients.jedis.tests.JedisTest#checkCloseable, redis.clients.jedis.tests.JedisTest#checkBinaryData, redis.clients.jedis.tests.JedisTest#checkDisconnectOnQuit, redis.clients.jedis.tests.JedisTest#startWithUrlString, redis.clients.jedis.tests.JedisTest#failWhenSendingNullValues, redis.clients.jedis.tests.JedisTest#useWithoutConnecting, redis.clients.jedis.tests.JedisTest#shouldNotUpdateDbIndexIfSelectFails, redis.clients.jedis.tests.JedisTest#allowUrlWithNoDBAndNoPassword, redis.clients.jedis.tests.JedisTest#connectWithShardInfo, redis.clients.jedis.tests.JedisTest#shouldThrowInvalidURIExceptionForInvalidURI], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=14}
status: NOPATCH
dateEnd: Sun Dec 03 08:46:49 EST 2017
allocatedtime: 105minutes 
passingTime: 1minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/310839695/src/main/java], projectClasspath='[Ljava.net.URL;@3269ab9b', projectTests=[redis.clients.jedis.tests.JedisTest], complianceLevel=8, outputFolder=./, json=false}
exception: null
nbStatements: 32
nbAngelicValues: 26
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #19
location: FailureLocation{className='redis.clients.jedis.tests.ShardedJedisPoolTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.ShardedJedisPoolTest#checkFailedJedisServer, redis.clients.jedis.tests.ShardedJedisPoolTest#checkPoolRepairedWhenJedisIsBroken, redis.clients.jedis.tests.ShardedJedisPoolTest#checkJedisIsReusedWhenReturned, redis.clients.jedis.tests.ShardedJedisPoolTest#checkResourceIsCloseable, redis.clients.jedis.tests.ShardedJedisPoolTest#startWithUrlString, redis.clients.jedis.tests.ShardedJedisPoolTest#checkCloseableConnections, redis.clients.jedis.tests.ShardedJedisPoolTest#checkConnections, redis.clients.jedis.tests.ShardedJedisPoolTest#shouldNotShareInstances, redis.clients.jedis.tests.ShardedJedisPoolTest#returnResourceShouldResetState, redis.clients.jedis.tests.ShardedJedisPoolTest#checkPoolOverflow, redis.clients.jedis.tests.ShardedJedisPoolTest#startWithUrl, redis.clients.jedis.tests.ShardedJedisPoolTest#shouldReturnActiveShardsWhenOneGoesOffline, redis.clients.jedis.tests.ShardedJedisPoolTest#checkConnectionWithDefaultPort], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=13}
status: NOPATCH
dateEnd: Sun Dec 03 08:48:22 EST 2017
allocatedtime: 105minutes 
passingTime: 1minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/310839695/src/main/java], projectClasspath='[Ljava.net.URL;@21454c82', projectTests=[redis.clients.jedis.tests.ShardedJedisPoolTest], complianceLevel=8, outputFolder=./, json=false}
exception: null
nbStatements: 56
nbAngelicValues: 35
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #20
location: FailureLocation{className='redis.clients.jedis.tests.commands.ControlCommandsTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.commands.ControlCommandsTest#save, redis.clients.jedis.tests.commands.ControlCommandsTest#lastsave, redis.clients.jedis.tests.commands.ControlCommandsTest#bgrewriteaof, redis.clients.jedis.tests.commands.ControlCommandsTest#sync, redis.clients.jedis.tests.commands.ControlCommandsTest#info, redis.clients.jedis.tests.commands.ControlCommandsTest#configSet, redis.clients.jedis.tests.commands.ControlCommandsTest#debug, redis.clients.jedis.tests.commands.ControlCommandsTest#monitor, redis.clients.jedis.tests.commands.ControlCommandsTest#configGet, redis.clients.jedis.tests.commands.ControlCommandsTest#waitReplicas, redis.clients.jedis.tests.commands.ControlCommandsTest#bgsave, redis.clients.jedis.tests.commands.ControlCommandsTest#readonly], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=12}
status: NOPATCH
dateEnd: Sun Dec 03 08:49:29 EST 2017
allocatedtime: 104minutes 
passingTime: 1minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/310839695/src/main/java], projectClasspath='[Ljava.net.URL;@3b992e08', projectTests=[redis.clients.jedis.tests.commands.ControlCommandsTest], complianceLevel=8, outputFolder=./, json=false}
exception: null
nbStatements: 32
nbAngelicValues: 26
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #21
location: FailureLocation{className='redis.clients.jedis.tests.commands.BitCommandsTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.commands.BitCommandsTest#testBitfield, redis.clients.jedis.tests.commands.BitCommandsTest#bitpos, redis.clients.jedis.tests.commands.BitCommandsTest#bitposBinary, redis.clients.jedis.tests.commands.BitCommandsTest#bitOp, redis.clients.jedis.tests.commands.BitCommandsTest#bitOpNot, redis.clients.jedis.tests.commands.BitCommandsTest#setAndgetbit, redis.clients.jedis.tests.commands.BitCommandsTest#testBinaryBitfield, redis.clients.jedis.tests.commands.BitCommandsTest#bitCount, redis.clients.jedis.tests.commands.BitCommandsTest#bitposWithNoMatchingBitExist, redis.clients.jedis.tests.commands.BitCommandsTest#setAndgetrange, redis.clients.jedis.tests.commands.BitCommandsTest#bitposWithNoMatchingBitExistWithinRange, redis.clients.jedis.tests.commands.BitCommandsTest#bitOpNotMultiSourceShouldFail], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=12}
status: NOPATCH
dateEnd: Sun Dec 03 08:50:39 EST 2017
allocatedtime: 104minutes 
passingTime: 1minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/310839695/src/main/java], projectClasspath='[Ljava.net.URL;@2e8ae93c', projectTests=[redis.clients.jedis.tests.commands.BitCommandsTest], complianceLevel=8, outputFolder=./, json=false}
exception: null
nbStatements: 32
nbAngelicValues: 26
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #22
location: FailureLocation{className='redis.clients.jedis.tests.JedisSentinelTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.JedisSentinelTest#sentinelFailover, redis.clients.jedis.tests.JedisSentinelTest#sentinelSet, redis.clients.jedis.tests.JedisSentinelTest#sentinelRemove, redis.clients.jedis.tests.JedisSentinelTest#sentinel, redis.clients.jedis.tests.JedisSentinelTest#sentinelMonitor], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:26379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:26379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:26381', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:26379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:26379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:26379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:26379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:26379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:26379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:26379', isError=true}], nbFailures=0, nbErrors=10}
status: NOPATCH
dateEnd: Sun Dec 03 08:51:46 EST 2017
allocatedtime: 103minutes 
passingTime: 1minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/310839695/src/main/java], projectClasspath='[Ljava.net.URL;@549a4c80', projectTests=[redis.clients.jedis.tests.JedisSentinelTest], complianceLevel=8, outputFolder=./, json=false}
exception: null
nbStatements: 55
nbAngelicValues: 38
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #23
location: FailureLocation{className='redis.clients.jedis.tests.SSLJedisTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.SSLJedisTest#connectWithoutShardInfo, redis.clients.jedis.tests.SSLJedisTest#connectWithShardInfoAndEmptyTrustStore, redis.clients.jedis.tests.SSLJedisTest#connectWithShardInfoAndCustomHostnameVerifier, redis.clients.jedis.tests.SSLJedisTest#connectWithShardInfoAndCustomSocketFactory, redis.clients.jedis.tests.SSLJedisTest#connectWithShardInfoAndCustomHostnameVerifierByIpAddress, redis.clients.jedis.tests.SSLJedisTest#connectWithShardInfo, redis.clients.jedis.tests.SSLJedisTest#connectWithUrl, redis.clients.jedis.tests.SSLJedisTest#connectWithShardInfoByIpAddress], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=8}
status: NOPATCH
dateEnd: Sun Dec 03 08:51:57 EST 2017
allocatedtime: 103minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/310839695/src/main/java], projectClasspath='[Ljava.net.URL;@53062126', projectTests=[redis.clients.jedis.tests.SSLJedisTest], complianceLevel=8, outputFolder=./, json=false}
exception: null
nbStatements: 2
nbAngelicValues: 2
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #24
location: FailureLocation{className='redis.clients.jedis.tests.commands.GeoCommandsTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.commands.GeoCommandsTest#geodist, redis.clients.jedis.tests.commands.GeoCommandsTest#geoadd, redis.clients.jedis.tests.commands.GeoCommandsTest#georadiusByMember, redis.clients.jedis.tests.commands.GeoCommandsTest#geopos, redis.clients.jedis.tests.commands.GeoCommandsTest#georadiusByMemberBinary, redis.clients.jedis.tests.commands.GeoCommandsTest#georadius, redis.clients.jedis.tests.commands.GeoCommandsTest#geohash, redis.clients.jedis.tests.commands.GeoCommandsTest#georadiusBinary], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=8}
status: NOPATCH
dateEnd: Sun Dec 03 08:52:49 EST 2017
allocatedtime: 103minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/310839695/src/main/java], projectClasspath='[Ljava.net.URL;@5a8e55ed', projectTests=[redis.clients.jedis.tests.commands.GeoCommandsTest], complianceLevel=8, outputFolder=./, json=false}
exception: null
nbStatements: 32
nbAngelicValues: 26
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #25
location: FailureLocation{className='redis.clients.jedis.tests.commands.HyperLogLogCommandsTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.commands.HyperLogLogCommandsTest#pfmerge, redis.clients.jedis.tests.commands.HyperLogLogCommandsTest#pfmergeBinary, redis.clients.jedis.tests.commands.HyperLogLogCommandsTest#pfadd, redis.clients.jedis.tests.commands.HyperLogLogCommandsTest#pfaddBinary, redis.clients.jedis.tests.commands.HyperLogLogCommandsTest#pfcounts, redis.clients.jedis.tests.commands.HyperLogLogCommandsTest#pfcount, redis.clients.jedis.tests.commands.HyperLogLogCommandsTest#pfcountBinary], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=7}
status: NOPATCH
dateEnd: Sun Dec 03 08:53:38 EST 2017
allocatedtime: 103minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/310839695/src/main/java], projectClasspath='[Ljava.net.URL;@4776fe23', projectTests=[redis.clients.jedis.tests.commands.HyperLogLogCommandsTest], complianceLevel=8, outputFolder=./, json=false}
exception: null
nbStatements: 32
nbAngelicValues: 26
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #26
location: FailureLocation{className='redis.clients.jedis.tests.commands.SortingCommandsTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.commands.SortingCommandsTest#sort, redis.clients.jedis.tests.commands.SortingCommandsTest#sortAlpha, redis.clients.jedis.tests.commands.SortingCommandsTest#sortLimit, redis.clients.jedis.tests.commands.SortingCommandsTest#sortBy, redis.clients.jedis.tests.commands.SortingCommandsTest#sortStore, redis.clients.jedis.tests.commands.SortingCommandsTest#sortGet, redis.clients.jedis.tests.commands.SortingCommandsTest#sortDesc], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=7}
status: NOPATCH
dateEnd: Sun Dec 03 08:54:29 EST 2017
allocatedtime: 103minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/310839695/src/main/java], projectClasspath='[Ljava.net.URL;@a27c8c1', projectTests=[redis.clients.jedis.tests.commands.SortingCommandsTest], complianceLevel=8, outputFolder=./, json=false}
exception: null
nbStatements: 32
nbAngelicValues: 26
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #27
location: FailureLocation{className='redis.clients.jedis.tests.commands.VariadicCommandsTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.commands.VariadicCommandsTest#zrem, redis.clients.jedis.tests.commands.VariadicCommandsTest#hdel, redis.clients.jedis.tests.commands.VariadicCommandsTest#sadd, redis.clients.jedis.tests.commands.VariadicCommandsTest#rpush, redis.clients.jedis.tests.commands.VariadicCommandsTest#zadd, redis.clients.jedis.tests.commands.VariadicCommandsTest#lpush], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=6}
status: NOPATCH
dateEnd: Sun Dec 03 08:55:14 EST 2017
allocatedtime: 103minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/310839695/src/main/java], projectClasspath='[Ljava.net.URL;@6577e878', projectTests=[redis.clients.jedis.tests.commands.VariadicCommandsTest], complianceLevel=8, outputFolder=./, json=false}
exception: null
nbStatements: 32
nbAngelicValues: 26
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #28
location: FailureLocation{className='redis.clients.jedis.tests.JedisSentinelPoolTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.JedisSentinelPoolTest#customClientName, redis.clients.jedis.tests.JedisSentinelPoolTest#checkResourceIsCloseable, redis.clients.jedis.tests.JedisSentinelPoolTest#returnResourceShouldResetState, redis.clients.jedis.tests.JedisSentinelPoolTest#checkCloseableConnections, redis.clients.jedis.tests.JedisSentinelPoolTest#ensureSafeTwiceFailover], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='All sentinels down, cannot determine where is mymaster master is running...', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='All sentinels down, cannot determine where is mymaster master is running...', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='All sentinels down, cannot determine where is mymaster master is running...', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='All sentinels down, cannot determine where is mymaster master is running...', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='All sentinels down, cannot determine where is mymaster master is running...', isError=true}], nbFailures=0, nbErrors=5}
status: NOPATCH
dateEnd: Sun Dec 03 08:56:30 EST 2017
allocatedtime: 103minutes 
passingTime: 1minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/310839695/src/main/java], projectClasspath='[Ljava.net.URL;@4265a8bf', projectTests=[redis.clients.jedis.tests.JedisSentinelPoolTest], complianceLevel=8, outputFolder=./, json=false}
exception: null
nbStatements: 95
nbAngelicValues: 59
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #29
location: FailureLocation{className='redis.clients.jedis.tests.ShardedJedisPipelineTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.ShardedJedisPipelineTest#pipelineResponse, redis.clients.jedis.tests.ShardedJedisPipelineTest#testSyncWithNoCommandQueued, redis.clients.jedis.tests.ShardedJedisPipelineTest#pipeline, redis.clients.jedis.tests.ShardedJedisPipelineTest#pipelineResponseWithinPipeline, redis.clients.jedis.tests.ShardedJedisPipelineTest#canRetrieveUnsetKey], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=5}
status: NOPATCH
dateEnd: Sun Dec 03 08:57:21 EST 2017
allocatedtime: 102minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/310839695/src/main/java], projectClasspath='[Ljava.net.URL;@539ab9b', projectTests=[redis.clients.jedis.tests.ShardedJedisPipelineTest], complianceLevel=8, outputFolder=./, json=false}
exception: null
nbStatements: 46
nbAngelicValues: 30
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #30
location: FailureLocation{className='redis.clients.jedis.tests.ShardedJedisTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.ShardedJedisTest#tryShardingWithMurmure, redis.clients.jedis.tests.ShardedJedisTest#testAvoidLeaksUponDisconnect, redis.clients.jedis.tests.ShardedJedisTest#checkCloseable, redis.clients.jedis.tests.ShardedJedisTest#trySharding], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6380', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6380', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6380', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=4}
status: NOPATCH
dateEnd: Sun Dec 03 08:59:12 EST 2017
allocatedtime: 102minutes 
passingTime: 1minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/310839695/src/main/java], projectClasspath='[Ljava.net.URL;@53477143', projectTests=[redis.clients.jedis.tests.ShardedJedisTest], complianceLevel=8, outputFolder=./, json=false}
exception: null
nbStatements: 150
nbAngelicValues: 90
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #31
location: FailureLocation{className='redis.clients.jedis.tests.commands.ObjectCommandsTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.commands.ObjectCommandsTest#objectEncoding, redis.clients.jedis.tests.commands.ObjectCommandsTest#objectIdletime, redis.clients.jedis.tests.commands.ObjectCommandsTest#objectRefcount], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=3}
status: NOPATCH
dateEnd: Sun Dec 03 08:59:49 EST 2017
allocatedtime: 102minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/310839695/src/main/java], projectClasspath='[Ljava.net.URL;@2731fc01', projectTests=[redis.clients.jedis.tests.commands.ObjectCommandsTest], complianceLevel=8, outputFolder=./, json=false}
exception: null
nbStatements: 35
nbAngelicValues: 28
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #32
location: FailureLocation{className='redis.clients.jedis.tests.commands.ConnectionHandlingCommandsTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.commands.ConnectionHandlingCommandsTest#quit, redis.clients.jedis.tests.commands.ConnectionHandlingCommandsTest#binary_quit], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=2}
status: NOPATCH
dateEnd: Sun Dec 03 09:00:28 EST 2017
allocatedtime: 102minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/310839695/src/main/java], projectClasspath='[Ljava.net.URL;@64cf8420', projectTests=[redis.clients.jedis.tests.commands.ConnectionHandlingCommandsTest], complianceLevel=8, outputFolder=./, json=false}
exception: null
nbStatements: 32
nbAngelicValues: 26
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #33
location: FailureLocation{className='redis.clients.jedis.tests.TupleSortedSetTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.TupleSortedSetTest#testString, redis.clients.jedis.tests.TupleSortedSetTest#testBinary], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}, FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=2}
status: NOPATCH
dateEnd: Sun Dec 03 09:01:12 EST 2017
allocatedtime: 102minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/310839695/src/main/java], projectClasspath='[Ljava.net.URL;@565bc68f', projectTests=[redis.clients.jedis.tests.TupleSortedSetTest], complianceLevel=8, outputFolder=./, json=false}
exception: null
nbStatements: 32
nbAngelicValues: 26
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #34
location: FailureLocation{className='redis.clients.jedis.tests.ConnectionCloseTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.ConnectionCloseTest#connectIfNotConnectedWhenSettingTimeoutInfinite], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=1}
status: NOPATCH
dateEnd: Sun Dec 03 09:01:33 EST 2017
allocatedtime: 102minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/310839695/src/main/java], projectClasspath='[Ljava.net.URL;@2a97285f', projectTests=[redis.clients.jedis.tests.ConnectionCloseTest], complianceLevel=8, outputFolder=./, json=false}
exception: null
nbStatements: 19
nbAngelicValues: 18
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #35
location: FailureLocation{className='redis.clients.jedis.tests.commands.SlowlogCommandsTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.commands.SlowlogCommandsTest#slowlog], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=1}
status: NOPATCH
dateEnd: Sun Dec 03 09:02:09 EST 2017
allocatedtime: 102minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/310839695/src/main/java], projectClasspath='[Ljava.net.URL;@58b7740', projectTests=[redis.clients.jedis.tests.commands.SlowlogCommandsTest], complianceLevel=8, outputFolder=./, json=false}
exception: null
nbStatements: 32
nbAngelicValues: 26
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #36
location: FailureLocation{className='redis.clients.jedis.tests.ModuleTest', failingMethods=[], erroringMethods=[redis.clients.jedis.tests.ModuleTest#testModules], failures=[FailureType{failureName='redis.clients.jedis.exceptions.JedisConnectionException', failureDetail='Failed connecting to host localhost:6379', isError=true}], nbFailures=0, nbErrors=1}
status: NOPATCH
dateEnd: Sun Dec 03 09:02:48 EST 2017
allocatedtime: 102minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[/root/workspace/xetorthio/jedis/310839695/src/main/java], projectClasspath='[Ljava.net.URL;@4e960cc9', projectTests=[redis.clients.jedis.tests.ModuleTest], complianceLevel=8, outputFolder=./, json=false}
exception: null
nbStatements: 32
nbAngelicValues: 26
ignoreStatus: NOTHING_TO_IGNORE
----------



